### PR TITLE
Update example explanation

### PR DESCRIPTION
**Pull request by mozeran**
Pull request to solve [issue 971](https://github.com/swcarpentry/shell-novice/issues/971)

### CODE REVIEW

**Comment by gdevenyi** [comment]
Thanks, I like the changes, I just have some small adjustment to the language.

**Comment by gcapes** [comment]
@gdevenyi 
What would those adjustments be?

**Comment by gdevenyi** [comment]
Man I'm bad at hitting that "request changes" button....

**Comment by mozeran** [comment]
If we change it to "which then," it's no longer a complete sentence. If you prefer "which then", should it be this (with similar modification for the other one)? "This time, the shell runs `head` on `unicorn.dat` and pipes the first two lines to the `tail` command, which then prints the second line of `unicorn.dat`."

File: _episodes/05-loop.md [hunk 1]
```diff
@@ -86,11 +86,12 @@ and run the `head` command.
 The first time through the loop,
 `$filename` is `basilisk.dat`.
 The interpreter runs the command `head` on `basilisk.dat`,
-and then prints the
-first three lines of `basilisk.dat`.
+pipes the first two lines to the `tail` command, 
+and then prints the second line of `basilisk.dat`.
 For the second iteration, `$filename` becomes
 `unicorn.dat`. This time, the shell runs `head` on `unicorn.dat`
```
Comment (ngranda) [inline _episodes/05-loop.md]:
```suggestion
`unicorn.dat`. This time, the shell runs `head` on `unicorn.dat`, 
```

File: _episodes/05-loop.md [hunk 2]
```diff
@@ -86,11 +86,12 @@ and run the `head` command.
...
-and prints the first three lines of `unicorn.dat`.
+pipes the first two lines to the `tail` command, 
+and then prints the second line of `unicorn.dat`.
```
[the remark this answers is not in the document]
Comment (gdevenyi) [reply _episodes/05-loop.md]: > If we change it to "which then," it's no longer a complete sentence. If you prefer "which then", should it be this (with similar modification for the other one)? "This time, the shell runs head on unicorn.dat and pipes the first two lines to the tail command, which then prints the second line of unicorn.dat."

Sure, I agree.

**Pull request merged**